egs/tedlium/s5/local/chain/README

```diff
@@ -0,0 +1,31 @@
+These are instruction to reproduce the TEDLIUM models described in
+"Purely sequence-trained neural networks for ASR based on lattice-free
+MMI", by Povey et al.
+
+First run:
+
+./run.sh
+
+until the end of stage 7. (local/nnet/run_dnn.sh can be skipped.)
+
+Then run:
+
+local/chain/run_tdnn.sh
+
+to see results for a generic chain model (these were not the results
+shown in the paper).
+
+Only after that, you can run:
+
+local/chain/run_tdnn_more_ce.sh
+
+to repeat the experiment reported in the third column of table 4.
+(The second script depends on files produced by the first one by
+default. It could be modified so that they are independent.) This
+experiment uses more cross entropy regularization and more parameters.
+
+Results for each setup are posted at the top of each file.
+
+The rescored test WERs are the same (12.9%), and the second experiment
```

Comment:
if you say so... but the truth is that I ran the training twice for the setup we put in the paper (the second time was to ensure these scripts worked), and one time got 12.9% WER and the other got 12.8% WER. I am quite positive that this is just due to noise in doing stochastic gradient descent.